common/config/azure-pipelines/ci.yaml

```diff
@@ -1,21 +1,24 @@
 # iModel.js CI Build
 
-trigger:
-  schedules:
-  - cron: "0 0 * * *"
+trigger: none
+
+pr:
+  drafts: false
+  branches:
+    include:
+    - master
+    - releases/*
+  paths:
+    include:
+    - common/config/azure-pipelines/ci.yaml
```

Comment:
Wait, doesn't this mean the build will only trigger if we change that YAML file?

Reply:
Yes, I don't want this CI to be triggered for most PRs, only fork-based ones, as the fast-ci is triggered for branch-based PRs.  There isn't a good way to specify that in YAML so excluding most runs by default.

For all fork-based PRs, this ci.yaml will only be triggered if someone on the team leaves a comment such as, `/AzurePipelines run imodeljs.imodeljs`. This is supported through [comment triggers](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers).  Although I haven't actually tested it yet, hoping to do so today or tomorrow.

Reply:
OK.  Makes sense.  Still seems like there are other paths we may want to include even in that case (ci-core.yaml) for example.  And maybe a comment explaining that the PR triggers are more exceptions to the rule would be helpful?

Also, I think some general comments in these explaining the differences between the different pipelines might be nice... not sure the separation between ci and fast-ci, for example, is all that clear (they both just say "iModel.js CI Build")

Reply:
Sounds good.  I need to add a README plus additional information about the release process, in particular the version-bump.yaml job.

I'll try to write something up by the end of the day.  Mind if I do that in a separate PR?  I'd like to get these in to avoid the draft PR builds asap.

Reply:
Sure